packages/react-components/src/components/ImageOverlay.tsx

```diff
@@ -178,6 +183,7 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
       styles={{ main: focusTrapZoneStyle, scrollableContent: scrollableContentStyle }}
       isDarkOverlay={true}
     >
+      <Announcer polite={{ id: `${title}`, message: localeStrings.overlayVisibleMessage ?? altText }} />
```

Comment:
Do we want to introduce a new string for this? Just a call out if we are going have `altText` read out as a default we may want to document it so developers are clear about this behaviour